dist/css/style.css

```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css2?family=Shantell+Sans:ital,wght@0,300;0,400;0,700;1,800&display=swap");
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  width: 390px;
+  background-color: #012622;
+  color: #fdfffc;
+  font-family: "Shantell Sans", cursive;
+  height: 100vh;
+}
+body ul {
+  list-style: none;
+  padding: 0;
+}
+body button {
+  outline-color: #fdfffc;
+}
+
+footer div a {
+  font-weight: 300;
+  font-size: 0.7em;
+}
+
+header {
+  padding: 18px 8px;
+}
+header nav {
+  background-color: #003b36;
+  border-radius: 8px;
+}
+header nav .collapse ul a {
+  color: #fdfffc;
+}
+header nav .collapse ul a:hover {
+  color: #e98a15;
+}
+header nav .collapse .active {
+  color: #e98a15 !important;
+}
+header h1 {
+  font-weight: 700;
+  color: #e98a15;
+}
+header .button {
+  background-color: #e98a15;
+  border-color: #e98a15;
+}
+header .date-time {
+  margin-top: 65px;
+  padding: 28px 14px;
+  color: #007b71;
+}
+header .date-time .time {
+  display: flex;
+  justify-content: end;
+}
+
+section .display-off {
+  display: none !important;
+}
+section .display-on {
+  display: block !important;
+}
+section .list-book {
+  display: none;
+}
+section .add-book {
+  height: 55vh;
+}
+section .contact-book {
+  display: none;
+}
+
+footer {
+  padding: 18px 8px;
+}
+footer div {
+  padding: 12px 18px;
+  border-radius: 8px;
+  background-color: #003b36;
+}
+footer div h6 {
+  color: #e98a15;
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+    background-color: #012622;
+  }
+  header .date-time .time {
+    display: flex;
+    justify-content: start;
```

Comment:
- Great Job for styling it with beautiful UI and making it responseive👍👍 however your input fields are covering the hole and add button is aligned on the left Please update it to have a better design👌👌
<img width="1386" alt="Screenshot 2023-02-23 at 12 12 15 PM" src="https://user-images.githubusercontent.com/52010789/220866717-477e5830-5509-4467-a300-b01beac34d8f.png">
- Please also arrange the lists of books like stable as the given wireframe 👍
<img width="1379" alt="Screenshot 2023-02-23 at 12 12 44 PM" src="https://user-images.githubusercontent.com/52010789/220867469-bed681b4-6ed7-4137-bfa0-b800c0b17a9a.png">
given wireframe
![Screenshot 2023-02-23 at 12 11 37 PM](https://user-images.githubusercontent.com/52010789/220867942-854dbe3c-a93a-4857-ad6b-03fbd056e169.png)

Reply:
Thanks, Mr. @binhussen for your constructive comments.
We will consider and apply them.